lib/index.js

```diff
@@ -58,32 +55,33 @@ function getImportsToResolve (original, includePaths, transformers) {
   return imports
 }
 
-function createTransformersMap (transformers) {
+function createTransformersMap(transformers) {
   if (!transformers) {
     return {}
   }
 
   // return map of extension strings to transformer functions
   return transformers.reduce((extensionMap, transformer) => {
-    transformer.extensions.forEach((ext) => {
+    transformer.extensions.forEach(ext => {
       extensionMap[ext] = transformer.transform
     })
     return extensionMap
   }, {})
 }
 
-function getLoaderConfig (ctx) {
+function getLoaderConfig(ctx) {
   const defaults = {
     includePaths: [],
     data: '',
     transformers: [],
-    outputStyle: 'nested',
-    resolveURLs: true
+    resolveURLs: true,
+    sassOptions: {}
   }
   const options = utils.mergeDeep(defaults, loaderUtils.getOptions(ctx) || {})
   const includePaths = options.includePaths
   const basedir = ctx.rootContext || options.context || ctx.options.context || process.cwd()
   const transformers = createTransformersMap(options.transformers)
+  const implementation = options.implementation || require('sass')
```

Comment:
FYI – this doesn't work. I believe it fails to deep copy the reference to `sass` or `node-sass` from `options.implementation`